docs/installing_deis/gce.rst

```diff
@@ -95,43 +95,37 @@ Create a SSH key that we will use for Deis host communication:
 
 Create some persistent disks to use for ``/var/lib/docker``. The default root partition of CoreOS
 is only around 4 GB and not enough for storing Docker images and instances. The following creates 3
-disks sized at 32 GB:
+disks sized at 64 GB:
 
 .. code-block:: console
 
-    $ gcutil adddisk --zone us-central1-a --size_gb 32 cored1 cored2 cored3
-
-    Table of resources:
-
-    +--------+---------------+--------+---------+
-    | name   | zone          | status | size-gb |
-    +--------+---------------+--------+---------+
-    | cored1 | us-central1-a | READY  |      32 |
-    +--------+---------------+--------+---------+
-    | cored2 | us-central1-a | READY  |      32 |
-    +--------+---------------+--------+---------+
-    | cored3 | us-central1-a | READY  |      32 |
-    +--------+---------------+--------+---------+
+    $ gcloud compute disks create cored1 cored2 cored3 --size 64GB --type pd-ssd --zone us-central1-f
```

Comment:
That's probably the safest choice, or making `--type` an environment variable or arg to the script.